lib/cli.rb

```diff
@@ -0,0 +1,31 @@
+class CLI
+  def self.put_string(text)
```

Comment:
I believe you don't need `self` here.

Here is some reference
https://stackoverflow.com/questions/4699687/when-to-use-self-foo-instead-of-foo-in-ruby-methods

Once again, can we make sure somehow that our code is not violating Ruby style guide? 😉